README.md

```diff
@@ -113,3 +113,9 @@ external formats like virtual environments
 * [install.python-poetry.org](https://github.com/python-poetry/install.python-poetry.org): The official Poetry
 installation script
 * [website](https://github.com/python-poetry/website): The official Poetry website and blog
+
+## Sponsors
+
+Thanks to [JetBrains](https://www.jetbrains.com) for supporting us with licenses for their tools.
+
+[<img src="https://resources.jetbrains.com/storage/products/company/brand/logos/jetbrains.svg" width="150" alt="JetBrains logo." />](https://www.jetbrains.com)
```

Comment:
Please check the size of the logo at https://github.com/radoering/poetry/tree/jetbrains-support?tab=readme-ov-file#sponsors

Adequate? Larger? Smaller?

Reply:
Logo size looks good. Should we consider a table instead?

Reply:
Good idea, but if I do not miss anything, you cannot get rid of the header row and borders:

| | |
|---|---|
|[<img src="https://resources.jetbrains.com/storage/products/company/brand/logos/jetbrains.svg" width="150" alt="JetBrains logo." />](https://www.jetbrains.com) | Thanks to [JetBrains](https://www.jetbrains.com) for supporting us with licenses for their tools.|